app/src/main/java/uk/co/ribot/androidboilerplate/ui/main/RibotsAdapter.java

```diff
@@ -38,9 +45,25 @@ public RibotViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
     }
 
     @Override
-    public void onBindViewHolder(RibotViewHolder holder, int position) {
+    public void onBindViewHolder(final RibotViewHolder holder, int position) {
         Ribot ribot = mRibots.get(position);
-        holder.hexColorView.setBackgroundColor(Color.parseColor(ribot.profile.hexColor));
+        if (ribot.profile.avatar != null && !ribot.profile.avatar.isEmpty()) {
```

Comment:
I'm not sure loading the real avatar is really needed. Because this is a boilerplate I tried to keep the app specific code to the minimum so code is easier to understand and starting a project from this one is simpler.